### ISSUE TITLE
FkProducer.java:12: Fake Producer implementation

### ISSUE THREAD

**Issue by 0pdd** [eo-cqrs/eo-kafka#44]
The puzzle `10-404e50f4` from #10 has to be resolved: 

https://github.com/eo-cqrs/eo-kafka/blob/7c5f31add23e7a1e44b1e55bb58c087300bc3a72/src/main/java/org/eocqrs/kafka/fake/FkProducer.java#L12-L12

The puzzle was created by @h1alexbel on 24-Feb-23. 

Estimate: 45 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by h1alexbel** [comment]
The fake producer can be built using an in-memory XML sheet.
```xml
<?xml version="1.0" encoding="UTF-8"?>
<log>
  <metadata>
    <id>1</id>
    <keySerializer>org.apache.kafka.common.serialization.StringSerializer</keySerializer>
    <valueSerialzier>org.apache.kafka.common.serialization.StringSerializer</valueSerialzier>
  </metadata>
  <chunks>
    <chunk>
      <id>1</id>
      <data>
        <topic>name</topic>
        <partition>1</partition>
        <key>key-0</key>
      </data>
    </chunk>
    <chunk>
      <id>2</id>
      <data>
        <topic>name2</topic>
        <partition>3</partition>
        <key>key-1</key>
      </data>
    </chunk>
  </chunks>
</log>
```
On each ```send``` method execution new chunk will be added. But where to store this file with initial values? So far, I don't know

**Comment by h1alexbel** [comment]
https://github.com/jcabi/jcabi-github/blob/master/src/main/java/com/jcabi/github/mock/MkStorage.java

**Comment by 0pdd** [comment]
@0pdd 3 puzzles [#292](https://github.com/eo-cqrs/eo-kafka/issues/292), [#293](https://github.com/eo-cqrs/eo-kafka/issues/293), [#364](https://github.com/eo-cqrs/eo-kafka/issues/364) are still not solved.

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#292](https://github.com/eo-cqrs/eo-kafka/issues/292), [#293](https://github.com/eo-cqrs/eo-kafka/issues/293) are still not solved; solved: [#364](https://github.com/eo-cqrs/eo-kafka/issues/364).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#373](https://github.com/eo-cqrs/eo-kafka/issues/373) is still not solved; solved: [#292](https://github.com/eo-cqrs/eo-kafka/issues/292), [#293](https://github.com/eo-cqrs/eo-kafka/issues/293), [#364](https://github.com/eo-cqrs/eo-kafka/issues/364).

**Comment by 0pdd** [comment]
@0pdd all 4 puzzles are solved here: [#292](https://github.com/eo-cqrs/eo-kafka/issues/292), [#293](https://github.com/eo-cqrs/eo-kafka/issues/293), [#364](https://github.com/eo-cqrs/eo-kafka/issues/364), [#373](https://github.com/eo-cqrs/eo-kafka/issues/373).
